settings_dialog.js

```diff
@@ -101,14 +107,70 @@ function onSettingsChanged(){
     }
 }
 
+function installedMessageBox(){
+    if(insDirs.installedDir != null && settings.installDir != insDirs.installedDir){
+        dialog.showMessageBox(win, messageOptions, (response) => {
+            if(response == 0){
+                moveInstalledVersion();
+                console.log("moving file...");
+            }
+            if(response == 1){
+                insDirs.installedDir = getInstallPath();
+                saveInstalledDir();
+            }
+        })
+    }
+}
+
 let browseFilesButton = document.getElementById("browseFilesButton");
 
 browseFilesButton.addEventListener("click", function(event){
-    const target = installPath;
+    const target = getInstallPath();
     console.log("Opening item:", target);
     shell.openItem(target);
 });
 
+let selectInstallLocation = document.getElementById("selectInstallLocation");
+
+const messageOptions = {
+    type: "warning",
+    buttons: ["Yes", "No"],
+    title: "Warning!",
+    message: "A Thrive installation folder already exists. Do you want to move all of the \n installed files into the new directory?",
+}
+
+selectInstallLocation.addEventListener("click", function(event){
+    dialog.showOpenDialog(win,
+    {
+        properties: ['openDirectory', 'promptToCreate']
+    }, 
+    function(path) {
+        if(path == undefined){
+            console.log("No folder selected");
+        }
+        else {
+            setInstallPath(String(path));
+            onSettingsChanged();
+            updateInstalledVersions();
+
+            currentInstallDir.innerHTML = "Directory: " + getInstallPath();
+
+            installedMessageBox();
+        }
+    });
+});
+
+let resestInstallLocation = document.getElementById("resetInstallLocation");
+
+resetInstallLocation.addEventListener("click", function(event){
```

Comment:
This function should not do anything (and the button could be disabled) if the current install location is the default one.